ndbench-core/src/main/java/com/netflix/ndbench/core/defaultimpl/NdBenchGuiceModule.java

```diff
@@ -62,4 +65,10 @@ IConfiguration getIConfiguration(ConfigProxyFactory proxyFactory) {
         return proxyFactory.newProxy(IConfiguration.class);
     }
 
+    @Provides
+    @Singleton
+    IEsConfig getEsNfndbenchConfig(ConfigProxyFactory factory) {
```

Comment:
Similarly to above, it would be preferable to not mention any ES specific configurations in the `ndbench-core` code either. Everything should be contained into the ES Plugin. If we can have a generic configuration for all data stores then that is fine.